cubi_tk/isa_tab/__init__.py

```diff
@@ -13,6 +13,57 @@
     Given a germline DNA sequencing ISA-tab file and a PED file, add new lines to the ISA-tab
     file and update existing ones, e.g., for newly added parents.
 
+``annotate``
+    Add annotation to an ISA-tab file, given a tsv file.
+
+
+Annotate
+--------
+
+``cubi-tk isa-tab annotate`` updates material and files nodes in ISA-tab studies and assays with annotations provided as
+tab-separated text file.
+
+In the annotation file header, target node types need to be indicated in ISA-tab style (i.e. "Source Name", etc.) while
+annotations are just named normally. Annotations for materials are automatically recorded as Characteristics, while
+annotations for files are recorded as Comments. Different node types can be annotated using only one annotation file, as
+demonstrated in the example below.
+
+By default, if Characteristics or Comments with the same name already exist for a node type, only empty values are
+updated. Overwriting existing values requires confirmation (`--force-update`).
+
+Annotations are only applied to only one study and assay, since material names are not necessarily unique between the
+same material types of different studies or different assays (and thus, annotations couldn't be assigned unambiguously).
+By default the first study and assay listed in the investigation file are considered for annotation. A specific study
+and assay may be selected by file name (not path!) via `--target-study` or `--target-assay`, resp.
+
+Example execution:
+``cubi-tk isa-tab annotate investigation.tsv annotation.tsv --target-study s_study.tsv --target-assay a_assay.tsv``
```

Comment:
I'm not sure I understand this example. Which files are local? How one would find out the `--target-study` file name?
Also, do I need to be in a specific directory to call this command?